src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1145,6 +1148,7 @@ private sealed class LoaderHolder
             // Verify that the current schema-defining arguments are default.
             // Get settings just for core arguments, not everything.
             string tmp = CmdParser.GetSettings(host, options, new ArgumentsCore());
+            tmp = Regex.Replace(tmp, @"[(sparse=\+)|(quote\+)]", "");
```

Comment:
I don’t understand what this is for... can you explain?

Reply:
The following code may throw if tmp is not empty (Line 1163) Thus, we need to remove those values here.

---
In reply to: [258307862](https://github.com/dotnet/machinelearning/pull/2630#discussion_r258307862) [](ancestors = 258307862)

Reply:
With changes made (probably) @artidoro, I have dropped the new regexp.